02_classify_mutations/run_mutation_classification.py

```diff
@@ -130,11 +130,6 @@ def process_args():
         'shuffle_labels',
         'skip_reason'
     ]
-    if io_args.log_file.exists() and io_args.log_file.is_file():
-        log_df = pd.read_csv(io_args.log_file, sep='\t')
-    else:
-        log_df = pd.DataFrame(columns=log_columns)
-        log_df.to_csv(io_args.log_file, sep='\t')
```

Comment:
Just double checking to make sure I understand this change, the code used to unnecessarily read in the logs here, then overwrite them with `log_df = None` on line 157?

Reply:
Not quite - the code here used to read in the whole existing log file, then [the code here](https://github.com/greenelab/mpmp/blob/4d77451e72339116e8acf75f23fa078ad88dbeab/mpmp/utilities/file_utilities.py#L230) would append the existing log file to the existing log file (or in other words it would duplicate it with some additional rows). Now I create a new `log_df` for each time the script is run, then I append it to the existing log file if it exists, which was my intention the first time around.

Not sure why past me thought this code was a good idea, but getting rid of it seems to achieve the original goal.